samples/snippets/detect_intent_texts_with_location.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google LLC
```

Comment:
is this copied or new, if so this should be rev'd

Reply:
updated to 2021.